src/index.js

```diff
@@ -126,8 +127,8 @@ app.get("/users/:username", async (req, res) => {
                 ),
             },
             analyses: {
-                recent: await recentAnalysesData(db, username, limit),
-                running: await runningAnalysesData(db, username, limit),
+                recent: await recent,
+                running: await running,
```

Comment:
I agree with @ianmcorvidae and I think the extra `{analyses: }` wrappers here aren't necessary and will make the UI code more cluttered, since we'll have to access `data.analyses.recent.analyses` instead of just `data.analyses.recent`.

Could we update this to extract the `analyses` arrays?
Maybe something like this (though I haven't tested out syntax like this yet):
```suggestion
                recent: (await recent)?.analyses,
                running: (await running)?.analyses,
```

Reply:
This worked 🎉